library/src/test/java/com/chuckerteam/chucker/internal/support/JsonConverterTest.kt

```diff
@@ -1,60 +1,14 @@
 package com.chuckerteam.chucker.internal.support
 
 import com.google.common.truth.Truth.assertThat
-import java.text.DateFormat
-import java.util.Date
-import java.util.Locale
-import org.junit.jupiter.api.Test
+import org.junit.Test
 
 class JsonConverterTest {
 
     @Test
     fun testInstance_sameInstance() {
```

Comment:
This test can also go as it's not really valuable (it's testing the `by lazy` implementation of Kotlin).

Reply:
Agreed. Deleted `JsonConverterTest`.